concepticondata/references/references.bib

```diff
@@ -3493,4 +3493,13 @@ @misc{Tjuka2020
   Year                                = {2020},
   Doi                                 = {10.5281/zenodo.4058506},
   Url                                 = {https://doi.org/10.5281/zenodo.4058506}
-}
\ No newline at end of file
+}
+
+@Book{Gabelentz1892,
+  Title                               = {Handbuch zur Aufnahme fremder Sprachen},
+  Author                              = {Gabelentz, Georg von der},
+  Year                                = {1892},
+  Publisher                           = {ES Mittler und Sohn},
+  Address                             = {Berlin},
```

Comment:
The comma should be removed after {Berlin}.

Reply:
Sorry, I missed this. Will implement it with the next commit!